lib/modules/manager/maven-wrapper/artifacts.spec.ts

```diff
@@ -0,0 +1,260 @@
+import type { Stats } from 'fs';
+import os from 'os';
+import type { StatusResult } from 'simple-git';
+import { envMock, mockExecAll } from '../../../../test/exec-util';
+import { env, fs, git, mockedFunction, partial } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import { resetPrefetchedImages } from '../../../util/exec/docker';
+import { getPkgReleases } from '../../datasource';
+import { updateArtifacts } from '.';
+
+jest.mock('../../../util/fs');
+jest.mock('../../../util/git');
+jest.spyOn(os, 'platform').mockImplementation(() => 'darwin');
+jest.mock('../../../util/exec/env');
+jest.mock('../../datasource');
+
+function mockMavenFileChangedInGit(fileName = 'maven-wrapper.properties') {
+  git.getRepoStatus.mockResolvedValueOnce(
+    partial<StatusResult>({
+      modified: [`maven.mvn/wrapper/${fileName}`],
+    })
+  );
+}
+
+describe('modules/manager/maven-wrapper/artifacts', () => {
+  beforeEach(() => {
+    GlobalConfig.set({ localDir: './' });
```

Comment:
please set like here:
https://github.com/renovatebot/renovate/blob/a3587fdfcf0d54623de14fe07fc3505463a020a7/lib/modules/manager/gomod/artifacts.spec.ts#L47-L52

so we're more close to real run paths